src/aics-image-viewer/components/ControlPanel/index.tsx

```diff
@@ -100,6 +101,7 @@ export default function ControlPanel(props: ControlPanelProps): React.ReactEleme
           bordered={false}
           className="control-panel"
```

Comment:
Someday we'll come through and replace this `Card` component with something more sensible for its use... I think there's an issue out for that actually